File: easy/Mixed-Content/Mixed Content.cs
using System;
using System.IO;
using System.Collections.Generic;
using System.Text.RegularExpressions;

class Program
{
    static void Main(string[] args)
    {
        using (StreamReader reader = File.OpenText(args[0]))
        while (!reader.EndOfStream)
        {
            string line = reader.ReadLine();
            if (null == line)
                continue;
            // do something with line
            ShowLine(line);
        }
    }
    
    static void ShowLine(string line){
            string[] words = line.Split(',');
            Regex rgx = new Regex(@"\d+");
            int total = words.Length;
            string result = "";
            for(int i=0;i<total;i++){
                if(!rgx.IsMatch(words[i])) result += words[i] + ',';
            }
            if(result.Length>0)result = result.Substring(0,result.Length-1) + '|' ;
            for(int i=0;i<total;i++){
                if(rgx.IsMatch(words[i]))result += words[i] + ',';
            }
            Console.WriteLine(result.Substring(0,result.Length-1));
        }
}

File: easy/Mixed-Content/Mixed Content.java
import java.io.*;
public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            ShowLine(line);
        }
    }
    
    static void ShowLine(String line){
        String[] words = line.split(",");
        int total = words.length;
        String result = "";
        for(int i=0;i<total;i++){
            if(!words[i].matches("\\d+")) result += words[i] + ',';
        }
        if(result.length()>0)result = result.substring(0,result.length()-1) + '|' ;
        for(int i=0;i<total;i++){
            if(words[i].matches("\\d+"))result += words[i] + ',';
        }
        System.out.println(result.substring(0,result.length()-1));
    }
}
